### ISSUE TITLE
About the train loss of SCNN

### ISSUE THREAD

**Issue by lHou157** [harryhan618/SCNN_Pytorch#1]
![44b055b56ef7493006310906e91cd13](https://user-images.githubusercontent.com/32443777/55599595-15d71a80-578b-11e9-9cd0-d00a29f80ee6.jpg)
![fc2fc34aa543b6f8f47afb4b41bf0b0](https://user-images.githubusercontent.com/32443777/55599596-15d71a80-578b-11e9-8c60-bed123dad243.jpg)

Hello！I really appreciate your code and I learned a lot from it. But I have met a problem that When I run train.py, the loss of epoch 0 and epoch 1 is 0.591 and 0.369 respectively, epoch 2 is nan, so is epoch 3, 4 ..., I  couldn't get it. Have you ever met this problem? 
I am looking forward to your replying or contact me with e-mail: 1012638724@qq.com. Thanks again!

**Comment by harryhan618** [comment]
Well, there are several hyperparameters to be tuned. E.g., in `net.py`, the scale between background weight, seg_loss, exist_loss is not sure. Also, the hyperparameter of optimizer (lr etc.) requires trials. Smaller lr or using pretrained vgg16 from torchvision maybe keep away NAN.
Please pardon me since I have limited computing resources and I couldn't provid a well tested trained model weight. Later I would update this repository. Star it if you want to keep updated.

**Comment by lilyzhng** [comment]
@lHou157 Hi there, do you find a way to resolve the above issue?
